FFXIVClientStructs/FFXIV/Client/System/Framework/Framework.cs

```diff
@@ -65,11 +81,26 @@ public unsafe partial struct Framework {
     [MemberFunction("E8 ?? ?? ?? ?? 89 47 2C")]
     public static partial long GetServerTime();
 
-    public string UserPath {
-        get {
-            fixed (char* p = userPath) {
-                return new string(p);
-            }
+    [Obsolete("Use SavePath")]public string UserPath => SavePath;
```

Comment:
Why rename it? The launch parameter it is set by is called UserPath

Reply:
I did miss that and opted for a more descriptive name. Is reverted now as you are correct and we should adhere to SE's naming